JFRWebSocket.m

```diff
@@ -717,6 +725,46 @@ -(void)dealloc
 
 /////////////////////////////////////////////////////////////////////////////
 /////////////////////////////////////////////////////////////////////////////
+@interface JFRThread ()
```

Comment:
Yeah, I get it. It's a problem that is obviated by changing `+` to `-` in `sharedLoop` :)